front/components/assistant/conversation/AgentAction.tsx

```diff
@@ -11,5 +12,10 @@ export function AgentAction({ action }: { action: AgentActionType }) {
       </div>
     );
   }
+  if (isDustAppRunActionType(action)) {
+    <div className="pb-4">
+      <span className="text-red">TODO(spolu) DUST APP RUN ACTION</span>
+    </div>;
```

Comment:
I assume this is intended

Reply:
removed the ; but yeah intended for now :+1: